actionpack/lib/abstract_controller/rendering.rb

```diff
@@ -16,8 +16,22 @@ module Rendering
     extend ActiveSupport::Concern
     include ActionView::ViewPaths
 
-    # Normalizes arguments, options and then delegates render_to_body and
+    # Normalizes arguments, options and then delegates <tt>render_to_body</tt> and
     # sticks the result in <tt>self.response_body</tt>.
+    #
+    # The primary options are:
```

Comment:
@fakefarm actually, are you able to remove the documentation in both of the files, and instead link to that part of the guide? Would reduce duplication in the documentation. If you could also do a short review of that part of the guide, that's be awesome :)